libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.scss

```diff
@@ -166,7 +166,7 @@
     height: 170px;
     border: none;
     background-size: 100% 100%;
-    background-image: url(https://sky.blackbaudcdn.net/static/skyux-public-assets/1.0.0-beta.2/assets/images/colorpicker/saturation-lightness.webp);
+    background-image: url(https://sky.blackbaudcdn.net/static/skyux-public-assets/1.0.0-beta.3/assets/images/colorpicker/saturation-lightness.webp);
```

Comment:
@Blackbaud-ErikaMcVey - JFYI - `intl-tel-input` updated the flag images. We store these on the CDN from the `skyux-public-assets` repo. I updated these in country and phone field but also just updated everywhere that we referenced the old version in the CDN url.